man-roxygen/model-sample-args.R

```diff
@@ -0,0 +1,96 @@
+#' @param chains (positive integer) The number of Markov chains to run. The
+#'   default is 4.
+#' @param parallel_chains (positive integer) The _maximum_ number of MCMC chains
+#'   to run in parallel. If `parallel_chains` is not specified then the default
+#'   is to look for the option `"mc.cores"`, which can be set for an entire \R
+#'   session by `options(mc.cores=value)`. If the `"mc.cores"` option has not
+#'   been set then the default is `1`. This argument is available for
+#'   the `$sample()` method but not for the `$sample_mpi()` method.
```

Comment:
since both sample and sample_mpi now use this same file I just added a sentence to the arguments that sample_mpi doesn't support